tests/test_cpt.py

```diff
@@ -38,15 +38,15 @@ def test_predict(self):
 
         self.cpt.noise_ratio = 1
         self.cpt.MBR = 3
-        self.assertEqual(self.cpt.predict([['A'], ['A', 'B']]), ['B', 'D'])
+        self.assertEqual(self.cpt.predict([['A'], ['A', 'B']]), ['B', 'C'])
         # Test if predictions are the same with multi threading turned off
-        self.assertEqual(self.cpt.predict([['A'], ['A', 'B']], False), ['B', 'D'])
+        self.assertEqual(self.cpt.predict([['A'], ['A', 'B']], False), ['B', 'C'])
 
         self.cpt.MBR = 2
         self.assertEqual(self.cpt.predict([['A', 'B']]), ['C'])
 
         self.cpt.noise_ratio = 0.2
-        self.cpt.MBR = 1
```

Comment:
Regarding the definition of the `MBR`, we want to have 2 sequence to match here.